src/components/drive-container.tsx

```diff
@@ -126,7 +122,7 @@ export default function () {
                             treeItem.isSelected = false;
                             treeItem.children = treeItem.children || [];
                             treeItem.children.push({
-                                id: `${treeItem.id}/new-folder-${num}`,
+                                id: `${treeItem.id}/virtual-new-folder`,
```

Comment:
We still need to check if there isn't already a `virtual-new-folder` and add a number if needed right?